dfx.json

```diff
@@ -91,6 +91,7 @@
 			"candid": "https://github.com/dfinity/internet-identity/releases/download/release-2024-10-25/internet_identity.did",
 			"wasm": "https://github.com/dfinity/internet-identity/releases/download/release-2024-10-25/internet_identity_dev.wasm.gz",
 			"init_arg": "(opt record { captcha_config = opt record { max_unsolved_captchas= 50:nat64; captcha_trigger = variant {Static = variant {CaptchaDisabled}}}})",
+			"specified_id": "rdmx6-jaaaa-aaaaa-aaadq-cai",
```

Comment:
There are lots of canisters that include `--specified-id xxx` in their canister creation or deploy commands.  This is unnecessary as the canister ID can be specified in `dfx.json`.  If we keep on simplifying like this we can do `dfx deploy` without arguments, and that will deploy all the canisters.